validate.bat

```diff
@@ -1,4 +1,4 @@
 echo %1
-python excel_validator.py "example/example.yml" %1 "Example" "C:\temp" --errors=true
+python excel_validator.py  "example/example.yml" C:\Users\john.elmasry\excel_validator\example\excel.xlsx "Example" "." --errors=true
+pause
```

Comment:
now will be hardcoded vale to file name, btw orginal version with temp value also looks fishy